wp-content/themes/mrb-bct/page-jobs.php

```diff
@@ -38,9 +39,13 @@
 					// echo "</pre>"; 
 					?>
 					<li class="jobs-list-item" id="job-<?php echo $job->guid;?>">
-						<h2><a href="/jobs/<?php echo $job->guid; ?>"><?php echo $job->title;?></a></h2>
+						<h2><a href="/jobs/<?php echo $job->guid;?>"><?php echo $job->title;?></a></h2>
+						<h3><?php echo $job->content->Vacancy->JobTypes ?></h3>
+						<h4><?php echo $job->content->Vacancy->Location ?></h4>
+						<h5><?php echo $job->content->Vacancy->Salary ?></h5>
+						<h3><?php echo date("jS M Y", strtotime($job->content->Vacancy->ClosingDate));?></h3>
 						<p><?php echo substr($job->content->Vacancy->Description, 0, 100); ?>...</p>
```

Comment:
You indicated you had issues with redirects? Have you considered the security of this line in particular? Can you see any issues surrounding security that might be causing the redirect issue?